### PR TITLE
Adding -f to the file move command

### PR DESCRIPTION
**Pull request by digitaltom**
Adding -f to the file move command when moving the mirrored directory to its final location.

This fixes a problem with some filesystem setups, see bsc#1203171

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/294178/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 9ee791db9034f172eea77b0f99d5d53158af6c41 into dbf3b13be0f607e03635b64e0091b4a7b3fbc944)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/294499/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 9ee791db9034f172eea77b0f99d5d53158af6c41 into e54da1b4500fddaa684869a1204a9d425b11154d)

File: lib/rmt/mirror.rb [hunk 1]
```diff
@@ -191,7 +191,7 @@ def replace_directory(source_dir, destination_dir)
 
     FileUtils.remove_entry(old_directory) if Dir.exist?(old_directory)
     FileUtils.mv(destination_dir, old_directory) if Dir.exist?(destination_dir)
-    FileUtils.mv(source_dir, destination_dir)
+    FileUtils.mv(source_dir, destination_dir, force: '-f')
```
Comment (mssola) [inline lib/rmt/mirror.rb:194]: Nitpicking: looking at the documentation, wouldn't "force: true" also work? (in fact, the example on the documentation is passing a boolean value)

Comment (digitaltom) [reply lib/rmt/mirror.rb]: Yes, makes sense, changed. I think I copied that from the proposal in Bugzilla.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/294632/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged fcfebcbe322b69ea2934424b8fadae2ef11c46fc into e54da1b4500fddaa684869a1204a9d425b11154d)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/298811/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-run-tests. (Merged fcfebcbe322b69ea2934424b8fadae2ef11c46fc into 47f1685a38ffceb892469c9e640afc848b5e63b0)

**Review by mssola — APPROVED** [review]

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/298818/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 1e7344971926886014d8f8b1c4c008c6723f083f into 47f1685a38ffceb892469c9e640afc848b5e63b0)

**Pull request merged**